src/main/java/seedu/staffsnap/logic/commands/EditCommand.java

```diff
@@ -78,10 +78,17 @@ public CommandResult execute(Model model) throws CommandException {
         Applicant applicantToEdit = lastShownList.get(index.getZeroBased());
         Applicant editedApplicant = createEditedApplicant(applicantToEdit, editApplicantDescriptor);
 
+        // Check if edited applicant has the identity as another existing applicant
```

Comment:
Did you mean "if edited applicant has the **same** identity as another applicant"?